iocore/dns/DNS.cc

```diff
@@ -587,7 +616,7 @@ DNSHandler::startEvent_sdns(int /* event ATS_UNUSED */, Event *e)
   this->validate_ip();
 
   SET_HANDLER(&DNSHandler::mainEvent);
-  open_cons(&ip.sa, false, n_con);
+  open_cons(nullptr, false, n_con);
```

Comment:
I'm ok with this, but giving an explicit param is more clear then hiding it into `open_cons`.

Reply:
Just do the same things as L597-598.